src/runtime/test/config-test.ini

```diff
@@ -133,3 +133,14 @@ run = true
 [uri-resolver.http://localhost:8080]
 factory = partition_resolver_simple
 arguments = 127.0.0.1:8080
+
+[security]
+  krb5_keytab = /etc/pegasus_tst-pegasus.keytab
```

Comment:
These files may not exist on test environment, will the test cases fail?

Reply:
Yes, these configs is used for init kerberos. Or we should delete this unit test.
By the way, I prefer to delete the unit test